examples/async_hosted.php

```diff
@@ -45,9 +45,8 @@
     switch ($status_response->getStatus()) {
       case "completed":
         echo "Document available for public download at: " . $status_response->getDownloadUrl() . "\n";
-        $doc_response = $docraptor->getAsyncDoc($status_response->getDownloadId());
         $file = fopen("/tmp/docraptor-hosted-php.pdf", "wb");
-        fwrite($file, $doc_response);
+        fwrite($file, file_get_contents($status_response->getDownloadUrl()));
```

Comment:
Since DocRaptor is hosting the document for user's we probably don't need to write it to a file any more. It seems good to do this in our test files, but what if our examples just simply echoed the download url?

Reply:
Very good catch 👍